webhook/README.md

```diff
@@ -136,13 +137,47 @@ Webhook Formatters
 
 * default - The default formatter
 * slack - A POST formatted specifically for Slack
+* spinnaker - A POST formatted specifically for Spinnaker
 
 #### Using the slack format
 
 In order to work with Slack POST hooks, you need to add the [incoming-webhook app](https://api.slack.com/incoming-webhooks).
 This will generate a look with the format 'https://hooks.slack.com/services/######/######/#######' which you will use as
 the **url** in the configuration file. See the detailed sample configuration above.
 
+#### Using the spinnaker format
+
+In order to work with Spinnaker POST hooks, you need to enable spinnaker support and set the base url.
+This will generate a look with the format 'https://www.spinnaker.io/reference/artifacts/#format' which you will use as
+the **url** in the configuration file. See the detailed sample configuration below.
+
+```json
+{
+  "webhooks": {
+    "helm": {
+      "url": "SPINNAKER WEBHOOK URL for HELM",
+      "events": [
+        "storage.afterCreate"
+      ],
+      "path": "*.tgz",
+      "format": "spinnaker"
+    },
+    "docker": {
+      "url": "SPINNAKER WEBHOOK URL for Docker",
+      "events": [
+        "docker.tagCreated"
+      ],
+      "format": "spinnaker"
+    }
+  },
+  "debug": false,
+  "timeout": 15000,
+  "baseurl": "Artifactory base URL -- http://localhost:8081/artifactory",
+  "enablespinnakersupport": true
```

Comment:
I wonder if this flag is really necessary. Is there any advantage to leaving this off (or, is there any disadvantage to turning it on) when spinnaker isn't being used? If not, I don't see why the flag should exist; support should just be enabled all the time, like it is with the other formats.

Reply:
Agree with you. enablespinnakersupport flag is not required and removed.